ksql-engine/src/test/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactoryTest.java

```diff
@@ -118,13 +129,55 @@ public void shouldPickUpPrefixedSslConfig() {
 
     // When:
     final SchemaRegistryClient client =
-        new KsqlSchemaRegistryClientFactory(config, restServiceSupplier, sslFactory).create();
+        new KsqlSchemaRegistryClientFactory(config, restServiceSupplier, sslFactory,
+                                            schemaRegistryClientFactory).create();
+
 
     // Then:
     assertThat(client, is(notNullValue()));
     EasyMock.verify(restService);
   }
 
+
+  @Test
+  public void shouldPassBasicAuthCredentialsToSchemaRegistryClient() {
+    // Given
+    final Map<String, Object> schemaRegistryClientConfigs = ImmutableMap.of(
```

Comment:
This should probably include a config that _shouldn't_ be passed to the SR client.